src/server/graphql/models/Team/teamMutation.js

```diff
@@ -18,14 +19,15 @@ export default {
     },
     async resolve(source, {newTeam}, {authToken}) {
       // require cachedUserId in the input so an admin can also create a team
-      const userId = newTeam.leader.cachedUserId;
+      const userId = newTeam.leader[0].cachedUserId;
```

Comment:
great catch, yeah i had originally called it `teamMembers` just changed it to leader since i couldn't ever think of a scenario where you'd have more than 1.